woodwork/logical_types.py

```diff
@@ -306,6 +310,16 @@ class EmailAddress(LogicalType):
 
     primary_dtype = "string"
 
+    def validate(self, series, return_indices=False):
+        regex = r"\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}\b"
```

Comment:
should this reference the config `email_inference_regex` value?

Reply:
updated